scaleway/resource_lb_test.go

```diff
@@ -80,9 +89,39 @@ func TestAccScalewayLbLb_WithIP(t *testing.T) {
 				Config: `
 					resource scaleway_lb_ip ip01 {
 					}
+
+					resource scaleway_vpc_private_network pnLB01 {
+						name = "pn-with-lb-static"
+					}
+
+					resource scaleway_lb lb01 {
+					    ip_id = scaleway_lb_ip.ip01.id
+						name = "test-lb-with-static"
+						type = "LB-S"
+						release_ip = true
+						private_network {
```

Comment:
Could you try to have a test where you have two private networks? One where you have 2 private networks and then you delete one?

Reply:
And one where you have one private network initially and you add a second one in a second step.

Reply:
Only one private network is allowed for now.